e2e/playwright/testing-samples-loading.spec.ts

```diff
@@ -73,7 +68,7 @@ test.describe('Testing in-app sample loading', () => {
       await expect(warningText).toBeVisible()
       await confirmButton.click()
 
-      await expect(codeLocator).toContainText('// ' + newSample.title)
+      await editor.expectEditor.toContain('// ' + newSample.title)
       await expect(unitsToast('in')).toBeVisible()
     })
   })
```

Comment:
This test should have a `skip` or use the environment variable to skip, because it is supposed to test a web-only use case. It clearly wasn't specific enough to test it wasn't in desktop though 😆